sdk/synapse/Azure.Analytics.Synapse.Spark/src/autorest.md

```diff
@@ -22,4 +22,12 @@ directive:
   from: swagger-document
   where: $.parameters.Endpoint
   transform: $.format = "url"
-```
\ No newline at end of file
+```
+
+``` yaml
+directive:
+- from: swagger-document
+  where: $.parameters.LivyApiVersion
+  transform: >
+    $["x-ms-client-default"] = $.default;
```

Comment:
jus curious why this is needed

Reply:
It is needed to ensure the constructor sets a default parameter for `livyApiVersion` -- apparently the AutoRest behavior recently changed.

From @jianghaolu:

AutoRest recently changed the behavior for `default` to not send the value and rely on the service to "choose the default". `x-ms-client-default` does what default used to do.

Reply:
This was removed with an update to the swagger file directly.